Frontend/app/src/main/java/io/happybugs/happybugs/activity/SignInActivity.java

```diff
@@ -0,0 +1,127 @@
+package io.happybugs.happybugs.activity;
+
+import android.content.Context;
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Patterns;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import io.happybugs.happybugs.APIInterface.APIInterface;
+import io.happybugs.happybugs.R;
+import io.happybugs.happybugs.network.RetrofitInstance;
+import okhttp3.ResponseBody;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+
+public class SignInActivity extends AppCompatActivity {
+
+    private Context curContext;
+    private Button btnGoToSignUp;
+    private Button btnSignIn;
```

Comment:
I think there is no need to name variables this way. goToSignUp, signIn... seems simpler and intuitive.